provisioner/influx-series.js

```diff
@@ -36,6 +36,15 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
     // Both the bid and price will be the pre-safety factor number
     bid: base.stats.types.Number,
     price: base.stats.types.Number,
+    securityToken: base.stats.types.String,
+  },
+});
+
+// Store when a security token was retreived
+module.exports.securityTokenRetreived = new base.stats.Series({
+  name: 'AwsProvisioner.SecurityTokenRetreived',
+  columns: {
+    securityToken: base.stats.types.String,
```

Comment:
By:
> We don't even want to write an influxdb query for this.

I mean I can't imagine how we would compute durations by comparing ids from different influxdb queries... Isn't that going to cause a lot of subqueries (like one per row)?